library/src/scala/quoted/Quotes.scala

```diff
@@ -3807,6 +3874,14 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
         /** Case class or case object children of a sealed trait or cases of an `enum`. */
         def children: List[Symbol]
+
+        /** Type reference to the symbol usable in the scope of its owner. */
```

Comment:
This (and termRef) should be expanded to mention something like:
```suggestion
        /** Type reference to the symbol usable in the scope of its owner.
         *  To get a reference to a symbol from a specific prefix `tp`, use `tp.memberType(symbol)` instead.
         */
```

Reply:
Also `@pre symbol.isType returns true`

Reply:
Done